config/settings/common.py

```diff
@@ -45,7 +48,7 @@
 # Apps specific for this project go here.
 LOCAL_APPS = (
     # custom users app
-    'tuteria_application_test.users.apps.UsersConfig',
+    'tuteria_application_test.users',
```

Comment:
There was no need to touch this file, what was there before is a better way of registering an app with django. it doesn't affect anything if it was left alone.